### ISSUE TITLE
Sql Database Projects: Apply changes from db instance to database project

### ISSUE THREAD

**Issue by m4ttheweric** [microsoft/azuredatastudio#17828]
TL;DR... Need a way to apply changes from a db instance to the sql database project.

**Is your feature request related to a problem? Please describe.**
It seems we are unable to apply changes from a database instance back to a database project using schema compare.

In SSMS it's possible to run a schema compare and apply changes from a database instance to a sql database project. 

Currently, in ADS, when you choose schema compare between a sql project and a database instance, it only allows you to "Apply" when going in the direction of Project -> Database. It is greyed out when going in the direction of Database -> Project.

**Describe the solution or feature you'd like**
Oftentimes when we are developing features, a dev is iterating on design and will make changes to their local dev database. Once finalized, we need a way to apply those local dev database changes back to the sqlproj. This works on a windows machine that has access to SSMS, but for our Mac devs, there's no way to do this other than to make a backup of the database, restore it onto a windows machine, and run the schema compare using SSMS.

**Comment by kisantia** [comment]
duplicate of #13308. Work on adding this is in progress https://github.com/microsoft/azuredatastudio/pull/17738.

**Comment by dzsquared** [comment]
@m4ttheweric - I don't have a good eta quite yet, but we're eager to get it to insiders so you can use it!
The linked PR, #17738, isn't merged in yet as we're still doing some housekeeping on the changes. With folks intermittently out of the office this month it's hard to make an estimate on when it'll merge and land in insiders.  If you'd like to follow along with the team on that PR you're absolutely welcome to subscribe to the PR, might be interesting :)
